gamestonk_terminal/cryptocurrency/onchain/onchain_controller.py

```diff
@@ -849,6 +1383,14 @@ def print_help(self):
 
 Whale Alert:
     whales            check crypto wales transactions
+
+BitQuery (note: some commands execution can take > 10 seconds):
+    trades            last trades by dex or month
+    volp              daily volume for crypto pair
+    volt              token volume on DEXes
+    senders           unique ethereum addresses which made a transaction
+    mt                top traded crypto pairs on given decentralized exchange
+    spreads           bid, ask prices, average spread for given crypto pair
```

Comment:
So, one thing I try to do is have intuitive names that will suggest me straight the command I'm doing. In part, the command should make me think of the description. 

With that in mind I would have something along lines:
```
    lt                     last trades by dex or month
    dvcp               daily volume for crypto pair
     tv                   token volume on DEXes
     stran              specific transactions by ethereum addresses
     ttcp                top traded crypto pairs on given decentralized exchange
     baas               bid, ask and average spread for given crypto pair
```

What do you think?

Reply:
> I would improve the "tradeAmount" column here, and drop a row that doesn't has an exchange name. <img alt="Screenshot 2021-12-06 at 17 59 48" width="375" src="https://user-images.githubusercontent.com/25267873/144897752-a99d8bb2-0519-4068-83fa-6a6382dc80c2.png">
> 
> It says that default is `ETH` but when I do `volp` I get an error. You need to do `required="-h" not in other_args,'. Also here the column name `tradeAmountUSD` could be improved. <img alt="Screenshot 2021-12-06 at 18 01 20" width="1226" src="https://user-images.githubusercontent.com/25267873/144897960-851fc244-a0d3-4464-8dfd-628f6c74d73d.png">
> 
> Same as above for `volt`. In addition given we specify a coin I don't think we need that column in here. <img alt="Screenshot 2021-12-06 at 18 03 55" width="469" src="https://user-images.githubusercontent.com/25267873/144898316-72a1ff23-a75e-43c3-a810-bbb5c778d0dc.png">
> 
> I'm still testing the other ones.

Thanks for all your comments, I tried to not change column names coming from source data, but I can improve it. What do you mean when you talk about better column naming ? 
e.g. `tradeAmount` -> you mean to change camel case names, or to change it more to `volume` ?
If you can give me few sugesstions with better naming will be much easier to me, to change it.

Reply:
No, so I just mean something simple like "Trade Amount".

You don't even need to do this one by one, you can do the following:
```
import re

df.columns = [' '.join(re.findall('.[^A-Z]*', val)).capitalize() for val in df.columns]
```
![Screenshot 2021-12-07 at 12 39 33](https://user-images.githubusercontent.com/25267873/145030665-b86faacf-1569-4f1e-aef8-638f26ab749c.png)

Reply:
@DidierRLopes thanks for an examples. Just wonder, how it will behave if you have it as in `sortby` argparser parameter?

So if user will try to write e.g. `volt -t 10 -s  Trade Amount` it will fail, because argparser will split space into seperate arguments like `Trade Amount` = ["Trade", "Amount"]`, is there any way to change this behavior ?

Reply:
So I'm fine with keeping the `-s tradeAmount` and then we just improve the way the data frame is output. Basically just before displaying it to the user